### PR TITLE
Flake8 and Readme Update

### PR DESCRIPTION
**Pull request by ghost**
* Fix All flake8 warnings & Add .flake8
* Update Readme.md

Kindly Review @walterbender @chimosky @quozl .
Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
-dotsAndBoxes
-============
+# dotsAndBoxes
 
-A sugar game about the classic Dot and boxes
+## A sugar game about the classic Dot and boxes
```
Comment (chimosky) [inline README.md]: This shouldn't be a header.

**Comment by chimosky** [comment]
Reviewed 9b77e8f, looks good.

Make the requested change and I'll merge.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Done!

**Comment by chimosky** [comment]
Thank you!

**Pull request merged**